src/lib/features/feature-search/search-utils.ts

```diff
@@ -50,14 +50,19 @@ export const applyGenericQueryParams = (
     queryParams: IQueryParam[],
 ): void => {
     queryParams.forEach((param) => {
+        const isSingleParam = param.values.length === 1;
         switch (param.operator) {
             case 'IS':
             case 'IS_ANY_OF':
                 query.whereIn(param.field, param.values);
                 break;
             case 'IS_NOT':
             case 'IS_NONE_OF':
-                query.whereNotIn(param.field, param.values);
+                if (isSingleParam) {
+                    query.whereNot(param.field, param.values[0]);
+                } else {
+                    query.whereNotIn(param.field, param.values);
+                }
```

Comment:
Is this necessary? A list with a single list is still a lilt, right? So `is not in [a]` and `is not in [a, b]` should both be valid?

Reply:
Yes, it is because it is doing following sql query:

not in (x), and if you put in null for project , not in (null), then it will not work, because SQL things.